<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaserVisionSensorClass</class>
 <widget class="QWidget" name="LaserVisionSensorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LaserVisionSensor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;黑体&quot;;</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>631</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>相机采集画面</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_camera_display">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>480</y>
     <width>381</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnStartProcess">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>（抓图）焊缝识别</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="result_show">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="title">
       <string>处理结果数据</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_show_results">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>识别焊缝类型:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_weldType">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_show_results_2">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>焊缝初始点坐标:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_initialCoor">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_show_results_3">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>焊缝宽度:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_wledWidth">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_show_results_4">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>焊缝倾斜角度:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_weldAngle">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>1431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>490</y>
     <width>347</width>
     <height>456</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>相机操作</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="bntEnumDevices">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>查找设备</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bntOpenDevices">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>打开</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bntCloseDevices">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>关闭</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="DeviceNum">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>设备数量:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="bntExposureTime">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>曝光设置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ExposureTimeLine">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="imgGrab">
      <property name="title">
       <string>采集控制</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="rbnt_Continue_Mode">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>连续采集</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="rbnt_SoftTigger_Mode">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>单帧采集</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="bntStartGrabbing">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>采集开始</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bntStopGrabbing">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>采集停止</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string>抓图</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="bntScreenShot">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>抓图</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="bntSave_JPG">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>保存JPG</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bntSave_BMP">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>保存BMP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bntSave_PNG">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>保存PNG</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>711</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>处理结果</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_result_display">
      <property name="minimumSize">
       <size>
        <width>675</width>
        <height>450</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>675</width>
        <height>450</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>631</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>抓图</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_screenshot_display">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LaserVisionSensor.qrc"/>
 </resources>
 <connections/>
</ui>
